siliconcompiler/client.py

```diff
@@ -174,6 +175,14 @@ def request_remote_run(chip):
                    cwd=local_build_dir)
     upload_file = os.path.abspath(os.path.join(local_build_dir, 'import.tar.gz'))
 
+    # Print a reminder for public beta runs.
+    if default_server_name in remote_run_url:
+        chip.logger.warning("Your job will be uploaded to a public server for processing in 5 seconds.\n")
+        chip.logger.warning("WARNING: Please remember that the SiliconCompiler public servers are not intended to process proprietary intellectual property.")
+        chip.logger.warning("SiliconCompiler is not responsible for any proprietary intellectual property that may be uploaded.\n")
```

Comment:
It's hard to notice this text in the log, because it comes right after the local import step. The newlines add a break in the logger output which increases the odds that people will notice that they're supposed to read the message.

Reply:
I don't think they should be there. The timeout should serve as enough of a break to notice it

Reply:
Respectfully, I disagree. I know that the message is supposed to be there, and I have trouble noticing the message when it is printed as a block, even with the delay. It just feels like the import step is taking a bit longer.

Reply:
I think it's totally fair that, in order to be readable, warning messages should be short and to the point - i.e. one line.
However, I think the main sentence that should be prefixed with the `WARNING` message is actually the second sentence.

So either of the following two would be fine for me:
Option 1
`WARNING: Please remember that the SiliconCompiler public servers are not intended to process proprietary intellectual property.`
`WARNING: SiliconCompiler is not responsible for any proprietary intellectual property that may be uploaded.`

or

Option 2
`Please remember that the SiliconCompiler public servers are not intended to process proprietary intellectual property.`
`WARNING: SiliconCompiler is not responsible for any proprietary intellectual property that may be uploaded.`

Reply:
Okay, here is what the current message looks like. How should it be modified, if at all?

```
[...surelog import task messages....]
[ SYNTAX] : 0
[  ERROR] : 0
[WARNING] : 1
[   NOTE] : 5
| WARNING | job0  | import     | 0  | [WRN:PA0205] /[...filepath...].v:1: No timescale set for "heartbeat".
| INFO    | job0  | import     | 0  | Finished task in 0.76s
| WARNING | job0  | import     | 0  | Your job will be uploaded to a public server for processing in 5 seconds.

| WARNING | job0  | import     | 0  | Please remember that the SiliconCompiler public servers are not intended to process proprietary intellectual property.
| WARNING | job0  | import     | 0  | SiliconCompiler is not responsible for any proprietary intellectual property that may be uploaded.

| INFO    | job0  | import     | 0  | Your job's reference ID is: [uuid]```

Reply:
I think it's sufficient as is. We can always circle back later if we decide to update the language.